__tests__/FSStorage.test.js

```diff
@@ -0,0 +1,94 @@
+import {
+    getCacheFromPhone,
+    setModificationsCacheFromPhone
+} from '../lib/FSStorage';
+import FsLogger from '../lib/FsLogger';
+import AsyncStorage from '@react-native-community/async-storage';
+
+/// Data Mock , inspired from real case
+export const dataMock = {
```

Comment:
put mock in a specific folder `mock` nearby unit test files

Reply:
done